buddyboss/buddyboss-platform
# Fix - Video thumbnail should contain the wrapper rather then cover

### All Submissions:


Fixes #2816 

### How to test the changes in this Pull Request:

1. Go to Activity
2. Upload a portrait video
3. Check thumbnail

### Proof Screenshots or Video
https://prnt.sc/1qfgtyx

### Other information:
